.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: "[Type] Bug"
+assignees: ''
+
+---
+
+<!-- Thanks for contributing to Sensei! Pick a clear title ("Lesson: Show complexity in individual lessons") and proceed. -->
+
+#### Steps to Reproduce
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+#### What I Expected
+
+
+#### What Happened Instead
+
+
+#### PHP / WordPress / Sensei version
+
+
+#### Browser / OS version
+
+
+#### Screenshot / Video
+
+
+#### Context / Source
+<!-- Optional: share your unique context to help us understand your perspective. -->
+
+
+
+<!--
+PLEASE NOTE
+- These comments won't show up when you submit the issue.
+- Everything is optional, but try to add as many details as possible.
+
+Contributing docs:
+https://github.com/Automattic/sensei/blob/master/CONTRIBUTING.md
```

Comment:
I think we should remove the Contributing docs section since this repo doesn't have one.